src/wfs/src/main/java/org/geoserver/wfs/WFSException.java

```diff
@@ -116,6 +105,32 @@ else if (className.endsWith("TypeImpl")) {
         return this;
     }
 
+	private String determineDefaultLocator(Object request) {
```

Comment:
If you changed the visibility of this method you could write tests cases directly for it.